community/llvm-libunwind/APKBUILD

```diff
@@ -1,42 +1,41 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 # Maintainer: Jakub Jirutka <jakub@jirutka.cz>
 pkgname=llvm-libunwind
+pkgver=5.0.0
 _pkgname=libunwind
-pkgver=4.0.0
 _llvmver=${pkgver%%.*}
 pkgrel=0
 pkgdesc="LLVM version of libunwind library"
 url="http://llvm.org/"
 arch="all !ppc64le !s390x"
 license="BSD"
 depends_dev="!libunwind-dev"
-makedepends="cmake llvm$_llvmver-dev"
+makedepends="cmake llvm-dev>=$_llvmver"
 subpackages="$pkgname-dev"
-source="http://www.llvm.org/releases/$pkgver/$_pkgname-$pkgver.src.tar.xz
-	no-exec-stack.patch"
+source="https://www.llvm.org/releases/$pkgver/$_pkgname-$pkgver.src.tar.xz"
 builddir="$srcdir/$_pkgname-$pkgver.src"
 options="!check"
 
 build() {
-	cd "$builddir"
+	mkdir -p "$builddir/build"
+	cd "$builddir/build"
 
-	cmake \
-		-DCMAKE_BUILD_TYPE=MinSizeRel \
+	cmake .. \
 		-DCMAKE_INSTALL_PREFIX=/usr \
```

Comment:
add `-DCMAKE_BUILD_TYPE=Release` ?

Reply:
Not quite. Setting build type to `Release` mode tells cmake to use `CMAKE_*_FLAGS_RELEASE` while abuild's own `CFLAGS` in `CMAKE_C*_FLAGS` are ignored.

Reply:
That's why there's MinSizeRel. Why did you remove it?

Reply:
Because cmake's `CMAKE_C_FLAGS_MINSIZEREL="-Os -DNDEBUG"` aren't quite the same `CFLAGS` as provided by abuild:  `CMAKE_C_FLAGS="-Os -fomit-frame-pointer"`

Reply:
That's true, but it's more complicated in CMake... IIRC the default profile is debug. Have you verified that it doesn't add `-g` to flags...?

Reply:
I think you are right. adding back the `-DCMAKE_BUILD_TYPE=MinSizeRel` made the `llvm5-static` size reduce to 117MB instead of gigabytes. Good catch!

Reply:
![That’s what I do, I drink and I know things](https://static2.hypable.com/wp-content/uploads/2016/04/game-of-thonres-tyrion-lannister-i-drink-and-i-know-things.jpg)
(except I don’t drink)

(Related to CMake profile: http://lists.alpinelinux.org/alpine-devel/5877.html)